packages/rum-core/src/domain/assembly.ts

```diff
@@ -91,6 +91,7 @@ export function startRumAssembly(
             session: {
               plan: session.hasReplayPlan() ? RumSessionPlan.REPLAY : RumSessionPlan.LITE,
             },
+            browser_sdk_version: configuration.sdkVersion,
```

Comment:
I decided to set the value even if the bridge isn't present. It will allow us to check the same version attribute regardless of the webview context.

Reply:
> It will allow us to check the same version attribute regardless of the webview context.

only from this version though, so we probably won't use it, no?

Reply:
Yes, we will probably still use both the tag and the attribute to have the full picture. IYO, does it adds value to put it only on the webview context?

Reply:
it avoids to bloat events with unnecessary attributes

Reply:
okay